build/mtaircompressor/startup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source $HOME/.setup_sal_env.sh
+
+run_mtaircompressor.py $RUN_ARG $* &
```

Comment:
I don't think that the `$*` is needed here. If you want to pass on arguments then they need to be added to `RUN_ARG`.

Reply:
I need to pass either 1 or 2 (index) when running container. I belive RUN_ARG will be taken only from Dockerfile, not when starting container on Kubernets. But I deleted the $*, assuming somehow magically RUN_ARG would become 1 or 2 when starting different containers.

Reply:
You're right.